framework/src/multiapps/MultiApp.C

```diff
@@ -502,6 +502,7 @@ MultiApp::createApp(unsigned int i, Real start_time)
   // Update the MultiApp level for the app that was just created
   app->setMultiAppLevel(_app.multiAppLevel() + 1);
   app->setupOptions();
+  preRunInputFile();
```

Comment:
@friedmud Let me just do a quick explanation what I want to do with this function and why it's here. I have a master mesh nested in the sub app mesh. The transfers are such that each sub app element that's in a master app element must get the same procedure ID from the partitioner. For ensuring that I have an EmbeddedMeshPartitioner. However, the EmbeddedMeshPartitioner requires the master mesh as input and is part of the sub app. Therefore, the multiapp is really the only place I could put it as I have access to both master and sub app stuff. I programmatically add an action in this call back that sets the correct partitioner and its parameters.